### PR TITLE
Add details to logError in TokenRequestValidator

### PR DESCRIPTION
**Pull request by shaikatz**
**What issue does this PR address?**
#2392 

**Does this PR introduce a breaking change?**
No

**Please check if the PR fulfills these requirements**
- [x] The commit follows our [guidelines](https://github.com/IdentityServer/IdentityServer4/blob/dev/.github/CONTRIBUTING.md)

**Other information**:

### CODE REVIEW

File: src/IdentityServer4/Validation/TokenRequestValidator.cs [hunk 1]
```diff
@@ -701,38 +701,42 @@ private TokenRequestValidationResult Invalid(string error, string errorDescripti
 
         private void LogError(string message = null, params object[] values)
         {
+            var details = new TokenRequestValidationLog(_validatedRequest);
             if (message.IsPresent())
             {
                 try
                 {
-                    _logger.LogError(message, values);
+                    _logger.LogError(message + ", , request details: {details}", values, details);
```
Comment (leastprivilege) [inline src/IdentityServer4/Validation/TokenRequestValidator.cs]: why 2 commas?

**Comment by shaikatz** [comment]
@leastprivilege Fixed.

File: src/IdentityServer4/Validation/TokenRequestValidator.cs [hunk 2]
```diff
@@ -701,38 +701,42 @@ private TokenRequestValidationResult Invalid(string error, string errorDescripti
...
 
         private void LogInfo(string message = null, params object[] values)
         {
+            var details = new TokenRequestValidationLog(_validatedRequest);
             if (message.IsPresent())
             {
                 try
                 {
-                    _logger.LogInformation(message, values);
+                    _logger.LogInformation(message + ", , request details: {details}", values, details);
```
Comment (leastprivilege) [inline src/IdentityServer4/Validation/TokenRequestValidator.cs]: same here

**Comment by shaikatz** [comment]
@leastprivilege Fixed

**Comment by leastprivilege** [comment]
thanks!

**Pull request merged**